.github/workflows/test_compat.yml

```diff
@@ -16,16 +16,16 @@ jobs:
     with:
       runs-on: '["ubuntu-latest", ]'
       python-version: '["3.8", ]'
-      depends: cython==0.29.24 numpy==1.17.5 scipy==1.3.2 nibabel==3.0.0 h5py==3.0.0 tqdm
+      depends: cython==0.29.24 numpy==1.22.0 scipy==1.8 nibabel==4.0.0 h5py==3.0.0 tqdm
```

Comment:
I suppose it is due to this change @frheault. 

you can look at my PR #2715 to update those values.

Also, my PR #2715 for integrating pyproject.toml is almost done. So I suppose, it will help TRX

Reply:
Should I wait or copy the values in all yml files under workflow?